src/helpers.js

```diff
@@ -0,0 +1,26 @@
+import {
+  getScores,
+} from './scoresAPI.js';
+
+function clearScores(parentContainer) {
```

Comment:
- [ ] Kindly ensure the usage of arrow functions instead of the function keyword. This is a key requirement for this project and also an effort in adapting ES6 syntax, please amend accordingly for all your functions.